internal/resolution/variablesources/bundles_and_dependencies_test.go

```diff
@@ -222,15 +340,7 @@ func VariableWithID(id deppy.Identifier) func(vars []*olmvariables.BundleVariabl
 func CollectBundleVariableIDs(vars []*olmvariables.BundleVariable) []string {
 	ids := make([]string, 0, len(vars))
 	for _, v := range vars {
-		ids = append(ids, v.Identifier().String())
+		ids = append(ids, v.Bundle().Name)
```

Comment:
I think this one should still use IDs. Bundle names are not unique. We need to update assert to use new bundle variable IDs.